smoke-test/tests/cypress/cypress/integration/siblings/siblings.js

```diff
@@ -31,6 +31,15 @@ describe('siblings', () => {
 
   it('can view individual nodes', () => {
     cy.login();
+
+    const resizeObserverLoopErrRe = /^[^(ResizeObserver loop limit exceeded)]/
+    cy.on('uncaught:exception', (err) => {
+        /* returning false here prevents Cypress from failing the test */
+        if (resizeObserverLoopErrRe.test(err.message)) {
+            return false
+        }
+    })
```

Comment:
this error is virtual browser specific and is safe to skip: https://stackoverflow.com/questions/49384120/resizeobserver-loop-limit-exceeded